patches/components-safe_browsing-core-browser-db-v4_local_database_manager.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/safe_browsing/core/browser/db/v4_local_database_manager.cc b/components/safe_browsing/core/browser/db/v4_local_database_manager.cc
+index f840561217284b0a9eb3d3bfcb68f3b0b9032986..c46a7a4137db855a2881bf4581c93cdee46a1126 100644
+--- a/components/safe_browsing/core/browser/db/v4_local_database_manager.cc
++++ b/components/safe_browsing/core/browser/db/v4_local_database_manager.cc
+@@ -107,7 +107,7 @@ ListInfos GetListInfos() {
+                SB_THREAT_TYPE_BLOCKLISTED_RESOURCE),
+       ListInfo(kSyncAlways, "UrlBilling.store", GetUrlBillingId(),
+                SB_THREAT_TYPE_BILLING),
+-      ListInfo(kSyncOnChromeDesktopBuilds, "UrlCsdDownloadAllowlist.store",
++      ListInfo(kSyncOnDesktopBuilds, "UrlCsdDownloadAllowlist.store",
```

Comment:
i wonder if we need to change other entries with `kSyncOnChromeDesktopBuilds` ? Given that we are never chrome branded, so it is always false

Reply:
Most of the Chrome-branded ones are for features which we disable. So it's fine to leave the lists disabled.

The only one I'm not sure about and will need to investigate is `UrlSuspiciousSite`.

Reply:
But it's not related to the download protection feature and so it's not needed to resolve the false positive that @bsclifton discovered.